### DISCUSSION TITLE
an essay on why not having blackjack is a bad statement for the server

### DISCUSSION THREAD

**Discussion by GIVEMEAUSERNAMETHEFUCK** [Glazelf/NinigiBot#47] [Ideas]
to start this off, bots great, fucks up sometimes but its cool. ok so, HOW can you have a bot that can do coinflips, and (eventually) roulettes, but not include blackjack? every gambling bot has a blackjack option. if you wanna make your bot look cool and fun to use blackjacks the only thing that's gonna make it look like that. no one does coinflips, no one cares about that crap, WE NEED BLACKJACK. period. if you want the kids to have a real gambling addiction you have to add blackjack. and as a contributor to mr glazelfy (contributed like 11 dollars to his broke ass) i demand ATLEAST a start to the blackjack command. it would be so worth it.

- signed, lina, except apparently every fucking username in the world is taken so now im whatever my current usernames is

**Comment by Glazelf** [comment]
Closed since gambling is not allowed rip
